src/Mockingbird/UI/View/OptionsView.swift

```diff
@@ -15,13 +15,28 @@ public class OptionsView {
 
         Group {
 
-            Tree("Server Context", options: .defaultOpen) {
+            Tree("Setup", options: .defaultOpen) {
 
+                Text("Server Context")
                 RadioButtonGroup(state.contextTitles, selectedIndexState: state.currentContextIndex) { val in
 
                     ContextManager.shared.setContext(index: val)
                 }
                 NewLine()
+
+                Text("WORKING DIRECTORY: \(Default.Folder.workingDirectory)")
+                LargeButton("CHANGE DIRECTORY") {
+
+                    if let newDirectory = Util.chooseFolder() {
+
+                        Default.Folder.savedWorkingDirectory = newDirectory
+                        Default.Folder.workingDirectory = newDirectory
```

Comment:
can we move this set to be inside the savedWorkingDirectory?

Reply:
Moved!